addons/udes_stock/models/stock_picking.py

```diff
@@ -837,11 +831,11 @@ def batch_to_user(self, user):
         if PickingBatch.get_user_batches():
             raise ValidationError(_('User %s already has an in progress batch') % user.name)
 
-        if self.batch_id.id is False:
```

Comment:
Hmmm, looks like the the check did never fail otherwise it would have broke, right?